File: examples/functions/python/auto-infer-python-multi-deps/handler.py
import json


def _inferred_imports_marker():
    import flask  # noqa: F401
    import httpx  # noqa: F401
    import requests  # noqa: F401
    return "multi-deps"


def handler(event):
    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps(
            {
                "runtime": "python",
                "function": "auto-infer-python-multi-deps",
                "inference": "multiple-imports",
            },
            separators=(",", ":"),
        ),
    }

File: examples/functions/node/auto-infer-node-multi-deps/handler.js
module.exports.handler = async () => {
  if (false) {
    require("dayjs");
    require("nanoid");
    require("uuid");
  }

  return {
    status: 200,
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify({
      runtime: "node",
      function: "auto-infer-node-multi-deps",
      inference: "multiple-imports",
    }),
  };
};
